plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/test-or-live-account/index.tsx

```diff
@@ -60,6 +60,16 @@ const TestOrLiveAccountStep = () => {
 								</p>
 							</div>
 						</div>
+						{ currentStep?.errors &&
```

Comment:
If there are unhandled errors sent by the backend, always show them. It is better to inform the user and ourselves).